snuba/datasets/discover.py

```diff
@@ -0,0 +1,149 @@
+from abc import ABC
+from datetime import timedelta
+from typing import Mapping, Sequence
+
+from snuba.clickhouse.columns import ColumnSet
+from snuba.datasets.dataset import TimeSeriesDataset
+from snuba.datasets.dataset_schemas import DatasetSchemas
+from snuba.datasets.factory import get_dataset
+from snuba.datasets.schemas import Schema
+from snuba.query.extensions import QueryExtension
+from snuba.query.parsing import ParsingContext
+from snuba.query.project_extension import ProjectExtension, ProjectWithGroupsProcessor
+from snuba.query.query import Query
+from snuba.query.query_processor import QueryProcessor
+from snuba.query.timeseries import TimeSeriesExtension
+from snuba.request.request_settings import RequestSettings
+from snuba.util import is_condition
+
+EVENTS = 'events'
+TRANSACTIONS = 'transactions'
+TRANSACTIONS_ONLY_COLUMNS = [
+    'trace_id',
+    'span_id',
+    'transaction_name',
+    'transaction_hash',
+    'transaction_op',
+    'start_ts',
+    'start_ms',
+    'finish_ts',
+    'finish_ms',
+    'duration',
+]
+
+EVENTS_ONLY_COLUMNS = [
+    'group_id',
+    'transaction',
```

Comment:
What about the various promoted tag columns?

Reply:
Thanks, they should be added now